<?xml version="1.0"?>
<launch>
  <arg name="azure_key" default="" doc="The API key for Azure Cloud" />
  <include file="$(find ur_e_webots)/launch/ur5e_joint_limited.launch" />
  <include file="$(find custom_moveit_config)/launch/ur5_e_moveit_planning_execution.launch" />
<!--   <node pkg="bin_picking" type="moveit_interface_node.py" name="moveit_interface_node" /> -->
  <node pkg="bin_picking" type="picking_node.py" name="picking_node" />
  <node pkg="speech_to_text" type="speech_to_text_node.py" name="speech_to_text_node" args="-k $(arg azure_key)" />
  <!-- <node pkg="text_to_speech" type="tts_node.py" name="text_to_speech_node" /> -->
  <node pkg="text_to_speech" type="tts_node_azure.py" name="text_to_speech_node" args="-k $(arg azure_key)" />
</launch>